src/main/java/vg/civcraft/mc/civmodcore/locations/spatial/octrees/BaseOcTreeNode.java

```diff
@@ -0,0 +1,191 @@
+/*
+MIT License
+
+Copyright (c) 2020 psygate (psygate.github.com)
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+
+package vg.civcraft.mc.civmodcore.locations.spatial.octrees;
+
+import vg.civcraft.mc.civmodcore.locations.spatial.IIntBBox3D;
+import vg.civcraft.mc.civmodcore.locations.spatial.IntBBox3D;
+
+import java.util.*;
```

Comment:
We don't do * imports as specified in our style guide https://github.com/CivClassic/style-guide/blob/master/stylecheck.xml I can reformat it myself if this is non-trivial to fix with your setup.